src/server/server-dev.js

```diff
@@ -45,7 +46,10 @@ const server = app.listen(PORT, () => {
   console.log(`App listening to ${PORT}....`)
   console.log('Press Ctrl+C to quit.')
 })
-GunDBPublic.init(server, conf.gundbPassword, 'publicdb', conf.gunPublicS3).catch(e => {
-  console.error(e)
-  process.exit(-1)
+AdminWallet.ready.then(_ => {
```

Comment:
this code block repeats in both server-dev/prod.js so it could be moved as the separate function somewhere to the GUN-related files:

```javascript
import conf from 'path/to/server-config'
import AdminWallet from 'path/to/AdminWallet'

const { gunPublicS3 } = conf

export const initalizeGunWithPrivateKey = async () => AdminWallet.ready.then(() => {
  const { wallets, addresses: [walletId] } = AdminWallet
  const pkey = wallets[walletId].privateKey.slice(2)

  return  GunDBPublic.init(server, pkey, 'publicdb', gunPublicS3)
 }).catch(exception => {
    console.log('gun failed... quiting', exception)
    process.exit(-1)
 })
```

Reply:
done